src/HealthChecks.SmbCifs/DependencyInjection/SmbCifsHealthCheckStorageBuilderExtensions.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
```

Comment:
this PR is only for Gcp CloudStorage, please split it

Reply:
If you merge the PR304 then I can send just this one reflecting only the cloudstorage PR since I am creating the pull requests form comparing against the master right I am not fully aware how to split this one from my fork